css/amp.css

```diff
@@ -27,7 +27,7 @@ html {
   overflow-x: hidden !important;
 }
 html, body {
-  height: auto !important;
+  height: 100% !important;
```

Comment:
This was added to fix https://github.com/ampproject/amphtml/issues/2187. Is this resolved now?